src/WebSite/Bit.Platform.WebSite/Web/Shared/Urls.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Bit.Platform.WebSite.Web.Shared
+{
+    public static class Urls
+    {
+        public const string HomePage = "/";
+
+        public const string Projects = "/Projects";
+
+        public const string Github = "https://github.com/bitfoundation/bitplatform";
+
+        public const string GithubRepo = "https://github.com/bitfoundation/bitplatform";
+
+        public const string GithubIssues = "https://github.com/bitfoundation/bitplatform";
```

Comment:
GithubIssues link is [https://github.com/bitfoundation/bitplatform/issues](https://github.com/bitfoundation/bitplatform/issues)